app/models/search/bookmark_indexer.rb

```diff
@@ -11,7 +11,7 @@ def self.index_all(options = {})
     unless options[:skip_delete]
       options[:skip_delete] = true
       BookmarkableIndexer.delete_index
-      BookmarkableIndexer.create_index(shards: 18)
+      BookmarkableIndexer.create_index(shards: 66)
```

Comment:
It would be nice if this was in the config so we can change it in the future without code changes

Reply:
Done :)